ecs-cluster/variables.tf

```diff
@@ -12,7 +12,7 @@ variable "region" {
 
 variable "keycloak-image" {
   type        = string
-  default     = "ghcr.io/hic-infra/ecs-keycloak:1.0.0-beta.1"
+  default     = "ghcr.io/hic-infra/ecs-keycloak:1.0.0-beta.2"
```

Comment:
When this repo is tagged with `1.0.0-beta.2` the image will be built and available for use